client/src/styles/global.scss

```diff
@@ -60,6 +59,17 @@
 
 .j40-address-readability {
   display: inline-block;
-  line-height: 1.5 !important;
+  line-height: 1.5 !important;  // trussworks issue
   text-align: center;
 }
+
+// This should really be part of uswds and use $theme-step-indicator-segment-color-complete
+.j40-usa-process-list__item--complete {
+  &::before {
+    color: white;
+    background-color: #00a91c;
+    content: '✓'
+  }
+
+  border-left-color: #005ea2 !important;  // todo: fix
```

Comment:
Can you make a ticket for this so we can track, and ref the ticket in the comment?

Reply:
This is required because the Progress element doesn't really support color changing the line connecting progress (like the Steps element does). So, I it took a bigger hammer to make it work.